src/components/Ghost/PostCard/index.tsx

```diff
@@ -54,31 +53,20 @@ const GhostPostCard = (props: GhostPostCardProps) => {
             <CardImage isVisible={post?.feature_image?.length > 0}>
             {post?.feature_image
```

Comment:
post?.feature_image && <Componente/> y no tienes que devolver null en caso de que no esté simplemente no lo pones 😉

Reply:
Sep lo usaba en principio, pero después de leer esto uso siempre ternario ``https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx``

Reply:
Oh! kentcdodds! okey, ya lo leeré entonces 😉